jobs/neuromotion_webbackend_01.html

```diff
@@ -0,0 +1,32 @@
+---
+title: Web Back End Engineer
+company: Neuromotion Labs
+url: http://neuromotionlabs.com
+location: Cambridge, MA
+# Choose one of the following options
+contract: contract
+contact:
+    name: Trevor Stricker
+    email: jobs@neuromotion-labs.com
+created: !!timestamp '2017-07-19' 
+tags:
+  - python
+  - sql
+  - django 
+  - Boston
+  - Video Games
+---
+
+# Company
+
+Neuromotion Labs uses the power of video games to help kids 6-14 build emotional strength. We are building a scientifically validated platform of video games, paired with coaching and a structured program aimed to improve emotional regulation. Neuromotion is based in Boston, near South Station. The team is composed of researchers from Harvard Medical School and veterans of games such as NBA 2K and Bioshock. And we are looking for help.
+
+# Overview
+
+Our paying customer base is growing and our marketing efforts are expanding. We need help with our infrastructure to keep pace. We are rapidly building our web experience for parents – including but not limited to online customer registration, payment flows, internal customer management and dashboards. This engineer will work closely with our front and back end engineers, as well as the business team to help translate the ever-changing market requirements into live features.
+
+# Requirements
+• 5+ years writing deployed web code
```

Comment:
I don't know if these bullet characters will work. Classic Markdown only supported * and - for bullets.

Did you test this?